### PR TITLE
[Compute AG] adding all the azure AD OIDC based configuration missing steps and details for PCSUP-12002

### PR DESCRIPTION
**Pull request by priteshchandaliya**

### CODE REVIEW

File: compute/admin_guide/authentication/oidc.adoc [hunk 1]
```diff
@@ -164,7 +164,15 @@ You can only view the value of the secret when you create it. Copy and store the
...
-. Add a group claim and select the *Groups assigned to the application* option.
+. Add *Optional claims* of Token type - ID. Check mark *email* and *preferred_username* claims. While saving these claims, it will prompt to enable Microsoft Graph email permission, enable it. Azure OIDC application should provides these claims to the Prisma Cloud Console to authenticate and render user info.
+
+image::oidc_optional_claim.png[width=250]
+
+. Add Microsoft Graph - Delegated - API permissions for *email, openid, profile*. Azure OIDC Application requires these permissions for accessing the API as the signed-in user.
+
+image::oidc_api_permission.png[width=250]
+
+. Add above group claim and select the *Groups assigned to the application* option.
```
Comment (rcaballeromx) [inline compute/admin_guide/authentication/oidc.adoc]: @priteshchandaliya The steps above do not mention any groups, they are mentioned below. Do we need to add this step or should the step that follows needs to be modified?

**Comment by rcaballeromx** [comment]
@Pubs-MV The PR is ready for merging.

**Review by pfox1969 — CHANGES_REQUESTED** [review]
I don't think we need the image::oidc_client_secret.png in there. Because you can only copy the client secret's value at the time of creation. That screen cap is after the creation.

**Review by pfox1969 — APPROVED** [review]

**Review by rcaballeromx — APPROVED** [review]
Thank you @pfox1969 for your changes.
I removed the screenshot as requested too.
@Pubs-MV I verified that the content builds.

**Comment by Pubs-MV** [comment]
also related to PCSUP-10521

**Pull request merged**